conventions/templatetags/custom_filters.py

```diff
@@ -413,6 +413,15 @@ def negate(condition: bool):
     return not condition
 
 
+@register.filter
+def attribute(object, key):
+    """Gets an attribute of an object dynamically from a string key"""
+    if hasattr(object, str(key)):
+        return getattr(object, key)
+
+    return None
```

Comment:
Je crois que tu peux l'écrire comme ça :

```suggestion
getattr(object, key, None)
```